### PR TITLE
Implement New Localization Leaderboard

### PR DESCRIPTION
**Pull request by ImmaZoni**
**Description:**
This Pull Request introduces a new feature, the Translation Leaderboard, designed to acknowledge and showcase the contributions of our dedicated community of translators on our Crowdin localization project. The Translation Leaderboard will provide a dynamic and transparent way to display the efforts and impacts of each contributor, fostering a sense of community and healthy competition.

**Details:**
The leaderboard will be generated by processing data from Crowdin, including each translator's username, contribution count, and avatar. It will filter out blacklisted contributors and will be saved in a user-friendly JSON format.

**Objective:**
To recognize and appreciate the translators for their invaluable contributions to our project.
To encourage participation in our localization efforts by fostering a sense of community and friendly competition.
Implementation:
The feature is implemented using Node.js, leveraging libraries such as axios for API requests, csv-parse for processing CSV data, and crypto for generating secure hash keys.

This feature aims to foster a collaborative and appreciative environment for all our contributors, ensuring their efforts do not go unnoticed, and encouraging more users to participate in our localization project.

### CODE REVIEW

File: docs/community/translate.md [hunk 1]
```diff
@@ -7,11 +7,39 @@ keywords:
...
     - Subspace
     - Translate
+    - Localization
 ---
+import TranslationLeaderboard from '@site/src/components/TranslationLeaderboard/index.jsx';
 
 # Translation Guide
 
-This guide will show you how to provide translations for this documentation.
+Welcome to the Subspace Docs translation guide! This guide is here to help you contribute to our goal of making Subspace more accessible and inclusive by providing translations. Subspace Labs is driven by a vision of a decentralized and equitable future, and we believe that overcoming language barriers is crucial to achieving this vision.
```
Comment (EmilFattakhov) [inline docs/community/translate.md:16]: I wonder if that makes more sense to use Subspace Network or Subspace Labs here?

File: docs/community/translate.md [hunk 2]
```diff
@@ -7,11 +7,39 @@ keywords:
...
+
+## Mission and Vision
+
+Subspace Labs is inherently driven by the vision of a more equitable and decentralized future. We believe that to truly fulfill our vision, we need content that caters to the linguistic diversity of the global community. Thus, the Subspace Labs Translation Initiative is born.
+
+### Our mission
+1. Deliver translated versions of our content, empowering visitors worldwide to learn about Subspace Labs in their language.
```
Comment (EmilFattakhov) [inline docs/community/translate.md:23]: Here as well, Labs or Network?

File: .github/scripts/leaderboard.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
+const axios = require('axios');
+const fs = require('fs').promises;
+const path = require('path');
+const { parse } = require('csv-parse');
+const crypto = require('crypto');
+
+const CONFIG = {
```
Comment (EmilFattakhov) [inline .github/scripts/leaderboard.js:7]: I love how you made this config, with fallbacks, dev-purposed JSON files and necessary settings, clean code Justin!

File: .github/scripts/leaderboard.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+                }
+            });
+            parser.on('end', function () {
+                const filteredRecords = records.filter(record => {
+                    const usernameFromCsv = extractUsername(record.Name);
+                    const key = generateKey(usernameFromCsv);
+                    if (membersMapping[key]) {
+                        Object.assign(record, membersMapping[key]);
+                    }
+                    const isBlacklisted = blacklistedContributors.has(record.originalName); // Check if originalName is blacklisted
```
Comment (EmilFattakhov) [inline .github/scripts/leaderboard.js:150]: Good idea to check that!

**Review by EmilFattakhov — APPROVED** [review]
This looks great Justin!

I think we should create an issue to increase the guide's images in size as they are very small now, but let's keep it separate. The leaderboard is awesome!

File: docs/community/translate.md (hunk 1 above)
Comment (ImmaZoni) [reply docs/community/translate.md]: Thanks for catching, got it adjusted to Network 🙏

File: docs/community/translate.md (hunk 2 above)
Comment (ImmaZoni) [reply docs/community/translate.md:23]: Thanks for catching, got it adjusted to Network 🙏

**Pull request merged**